### PR TITLE
Code style improvements

### PR DESCRIPTION
**Pull request by fxedel**
Fixed a lot of issues from LGTM (#177).

### CODE REVIEW

**Comment by zekoz** [comment]
Repo-wide formatting-only changes make history harder to read.

**Comment by fxedel** [comment]
@zekoz So should I split this up into several PRs?

File: freeciv-web/src/main/webapp/javascript/client_main.js [hunk 1]
```diff
@@ -38,45 +38,47 @@ function set_client_state(newstate)
     civclient_state = newstate;
 
     switch (civclient_state) {
-    case C_S_RUNNING:
-      clear_chatbox();
-      $.unblockUI();
-      show_new_game_message();
+      case C_S_RUNNING:
```
Comment (zekoz) [inline freeciv-web/src/main/webapp/javascript/client_main.js:41]: Where possible follow https://github.com/freeciv/freeciv/blob/master/doc/CodingStyle#L76. No need for this indent.

**Comment by zekoz** [comment]
My preference is to only fix style in things you were changing anyway. Would you prefer to talk to the other developers in irc or discord so that you don't end up doing unnecessary work?

**Comment by mchenryc** [comment]
> should I split this up into several PRs

Yes, for sure. The best thing you can do is to keep commits and PRs contextual. Small PRs are great, as they're easy to review and approve. Large, multi-faceted PRs are more difficult to review, and frankly, may get rejected for one portion, though other portions would have been accepted if they were independent.

Did I mention how great small PRs are?

As zekoz mentions, PR should generally have a purpose other than"fixing code" - the thing with changing lines just for the sake of style or syntax changes is that it hides the previous "logic" change. In other words, the _why_ a line of codes existence, can usually be determined by the comments in the previous commit for that line and those around it - [git blame](https://www.atlassian.com/git/tutorials/inspecting-a-repository/git-blame) is used to see why a change was made, understand the context, and determine if, and how it can/should be modified. To change code simply for the sake of whitespace, or re-arranging lines is frowned upon, because it unnecessarily hides/buries history.

That said, personally I love deleting dead code and encourage it when possible! As long as it's _actually_ dead code, deletion doesn't change the history of the working code around it.

The exception to "don't change for the sake of" is when explicit cleanup tasks are warranted/desired. There is an [open issue](https://github.com/freeciv/freeciv-web/issues/177) to clean up code smells with many commits already accepted towards that end, and others [actively being worked on](https://github.com/freeciv/freeciv-web/pull/214). Such changes are welcome - reference the issue in your PR, and keep it on task.

**Comment by zekoz** [comment]
Thank you for giving that explanation. That should go into a doc in the repo almost verbatim.

**Comment by fxedel** [comment]
So what should I do now? Create small PRs for similar files? Or rather similar changes? Is it really worth creating new PRs if the changes just bury the history?

**Comment by andreasrosdalw** [comment]
@fxedel Please update this pull-request, so that the merge conflicts are fixed. Then I will accept it, and merge it into this repo. Thanks for your contribution.

**Comment by andreasrosdalw** [comment]
Please reopen this pull-request when the merge conflicts are resolved.

**Pull request closed without merging**